Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -300,6 +302,19 @@ public void SetOutputOptions(OutputOptions options)
 			get; private set;
 		}
 
+		public Assembly[] GetAssemblies() {
+			lock (assemblies) {
+				return assemblies.ToArray ();
+			}
+		}
+
+		public Assembly[] GetAssemblies(long processId)
```

Comment:
This method should be internal, since getting the assemblies for a process is done through the ProcessInfo object.